### PR TITLE
Download qcow2 CaaSP image with wilcard

### PR DESCRIPTION
**Pull request by mjura**

### CODE REVIEW

**Comment by mjura** [comment]
Small explanation, to this change:
- `wget` will download CaaSP image, which match with pattern `SUSE-CaaS-Platform-1.0-KVM-and-Xen.x86_64*qcow2` to `/tmp`
- `find` will get this image and move it to `$IMAGE_PATH` and rename it to generic name `SUSE-CaaS-Platform-1.0-KVM-and-Xen.x86_64.qcow2`

File: contrib/libvirt/k8s-libvirt.sh [hunk 1]
```diff
@@ -61,7 +61,8 @@ if [ "$1" == "apply" ]; then
             wget -O "$IMAGE_PATH" "http://download.opensuse.org/repositories/Virtualization:/containers:/images:/KVM:/Leap:/42.2/images/Base-openSUSE-Leap-42.2.x86_64-cloud_ext4.qcow2"
         elif [ "$FLAVOUR" == "caasp" ]; then
             echo "[+] Downloading SUSE CaaSP qcow2 VM image to '$IMAGE_PATH'"
-            wget -O "$IMAGE_PATH" "http://download.suse.de/ibs/SUSE:/SLE-12-SP2:/Update:/Products:/CASP10/images/SUSE-CaaS-Platform-1.0-KVM-and-Xen.x86_64-1.0.0-Build8.13.qcow2"
+            wget -r -l1 -nd  "http://download.suse.de/ibs/SUSE:/SLE-12-SP2:/Update:/Products:/CASP10/images/" -P /tmp -A "SUSE-CaaS-Platform-1.0-KVM-and-Xen.x86_64*qcow2"
```
Comment (jimmykarily) [inline contrib/libvirt/k8s-libvirt.sh]: I guess this only downloads one image right? I mean the `-A` argument should filter all the rest.

Comment (mjura) [inline contrib/libvirt/k8s-libvirt.sh]: Yes, this will download only image file and ignore all the rest

**Comment by jimmykarily** [comment]
The only problem I see is that there is no way to "freeze" the version, not even for a while. As soon as a new build is out the script will try to download it blocking the development in the meantime. I hope it won't be too big of a problem.
👍

**Comment by MaximilianMeister** [comment]
and don't worry about the e2e-tests for now, as they are broken for another reason

**Pull request merged**